packages/node_modules/@webex/internal-plugin-conversation/src/conversation.js

```diff
@@ -14,19 +14,29 @@ import uuid from 'uuid';
 import {InvalidUserCreation} from './convo-error';
 import ShareActivity from './share-activity';
 
+const idToUrl = new Map();
 
 const Conversation = WebexPlugin.extend({
   namespace: 'Conversation',
 
+  /**
+   * @param {Object} conversation
+   * @param {Object} object
```

Comment:
What is this object? Could you add a bit more info?

Reply:
I added this to remove the warnings, I can remove them if we don't want partial comments